### DISCUSSION TITLE
so-soc missing when issuing command so-status

### DISCUSSION THREAD

**Discussion by jamestest11** [Security-Onion-Solutions/securityonion#9700] [General] [answered]
I have security onion installed on Ubuntu.  Has been working fine for quite some time.  Came in this morning to find my security onion (standalone) inoperable after management NIC went offline yesterday.  Issued a so-status command to find so-soc was a status of MISSING.  I rebooted and this didn't fix it.  Performed upgrade via soup, this didn't fix it.  I also reinstalled Security Onion (latest build).  This also did not fix the issue.  Viewing the logs at /opt/so/log/soc, I get the following output:  

timestamp=2023-02-01T18:33:42.807798893Z level=info message="Version Information" buildTime=2023-01-12T07:17:58Z version=2.3.200
timestamp=2023-02-01T18:33:42.80801295Z level=info message="Adding new preprocessor" priority=0 type=*web.BasePreprocessor
timestamp=2023-02-01T18:33:42.808069458Z level=info message="Launching modules"
timestamp=2023-02-01T18:33:42.808089834Z level=info message="Initializing module" module=sostatus
timestamp=2023-02-01T18:33:42.808119254Z level=info message="Initializing module" module=statickeyauth
timestamp=2023-02-01T18:33:42.80815238Z level=info message="Adding new preprocessor" priority=100 type=*statickeyauth.StaticKeyAuthImpl
timestamp=2023-02-01T18:33:42.808184538Z level=info message="Initializing module" module=staticrbac
timestamp=2023-02-01T18:33:42.808210406Z level=info message="Initializing module" module=filedatastore
timestamp=2023-02-01T18:33:42.80837459Z level=error message="Syntax error reading json object" error="unexpected end of JSON input" offset=0
timestamp=2023-02-01T18:33:42.808402065Z level=error message="Unable to load job file" error="unexpected end of JSON input" file=jobs/onion/1040.json
timestamp=2023-02-01T18:33:42.808420405Z level=error message="Failed to initialize module" error="unexpected end of JSON input" module=filedatastore
timestamp=2023-02-01T18:33:42.808437718Z level=error message="Aborting module launch due to initialization error" error="unexpected end of JSON input"
timestamp=2023-02-01T18:33:42.808698943Z level=info message="Terminating Modules"
timestamp=2023-02-01T18:44:56.285638895Z level=info message="Version Information" buildTime=2023-01-12T07:17:58Z version=2.3.200
timestamp=2023-02-01T18:44:56.28583686Z level=info message="Adding new preprocessor" priority=0 type=*web.BasePreprocessor
timestamp=2023-02-01T18:44:56.285878473Z level=info message="Launching modules"
timestamp=2023-02-01T18:44:56.285890369Z level=info message="Initializing module" module=staticrbac
timestamp=2023-02-01T18:44:56.285917335Z level=info message="Initializing module" module=filedatastore
timestamp=2023-02-01T18:44:56.286074094Z level=error message="Syntax error reading json object" error="unexpected end of JSON input" offset=0
timestamp=2023-02-01T18:44:56.28610788Z level=error message="Unable to load job file" error="unexpected end of JSON input" file=jobs/onion/1040.json
timestamp=2023-02-01T18:44:56.286136744Z level=error message="Failed to initialize module" error="unexpected end of JSON input" module=filedatastore
timestamp=2023-02-01T18:44:56.286160507Z level=error message="Aborting module launch due to initialization error" error="unexpected end of JSON input"
timestamp=2023-02-01T18:44:56.286706454Z level=info message="Terminating Modules"
timestamp=2023-02-01T18:59:17.084019957Z level=info message="Version Information" buildTime=2023-01-12T07:17:58Z version=2.3.200
timestamp=2023-02-01T18:59:17.084256249Z level=info message="Adding new preprocessor" priority=0 type=*web.BasePreprocessor
timestamp=2023-02-01T18:59:17.084288861Z level=info message="Launching modules"
timestamp=2023-02-01T18:59:17.084298001Z level=info message="Initializing module" module=kratos
timestamp=2023-02-01T18:59:17.084310901Z level=info message="Adding new preprocessor" priority=110 type=*kratos.KratosPreprocessor
timestamp=2023-02-01T18:59:17.084328399Z level=info message="Initializing module" module=elastic
timestamp=2023-02-01T18:59:17.088415317Z level=info message="Initialized Elasticsearch Client" HostUrl=https://10.1.10.228:9200 Index=*:so-*,*:endgame-* InsecureSkipVerify=true Password=***** TimeoutMs=5m0s Username=so_elastic
timestamp=2023-02-01T18:59:17.088454465Z level=info message="Initializing module" module=influxdb
timestamp=2023-02-01T18:59:17.088786967Z level=info message="Initializing module" module=sostatus
timestamp=2023-02-01T18:59:17.088799355Z level=info message="Initializing module" module=statickeyauth
timestamp=2023-02-01T18:59:17.088814345Z level=info message="Adding new preprocessor" priority=100 type=*statickeyauth.StaticKeyAuthImpl
timestamp=2023-02-01T18:59:17.088829608Z level=info message="Initializing module" module=staticrbac
timestamp=2023-02-01T18:59:17.088847821Z level=info message="Initializing module" module=filedatastore
timestamp=2023-02-01T18:59:17.089688975Z level=error message="Syntax error reading json object" error="unexpected end of JSON input" offset=0
timestamp=2023-02-01T18:59:17.089711594Z level=error message="Unable to load job file" error="unexpected end of JSON input" file=jobs/onion/1040.json
timestamp=2023-02-01T18:59:17.089725593Z level=error message="Failed to initialize module" error="unexpected end of JSON input" module=filedatastore
timestamp=2023-02-01T18:59:17.089741713Z level=error message="Aborting module launch due to initialization error" error="unexpected end of JSON input"
timestamp=2023-02-01T18:59:17.089878095Z level=info message="Terminating Modules"
timestamp=2023-02-01T19:22:51.195895506Z level=info message="Version Information" buildTime=2023-01-12T07:17:58Z version=2.3.200
timestamp=2023-02-01T19:22:51.196115019Z level=info message="Adding new preprocessor" priority=0 type=*web.BasePreprocessor
timestamp=2023-02-01T19:22:51.196158921Z level=info message="Launching modules"
timestamp=2023-02-01T19:22:51.196170416Z level=info message="Initializing module" module=filedatastore
timestamp=2023-02-01T19:22:51.197130792Z level=error message="Syntax error reading json object" error="unexpected end of JSON input" offset=0
timestamp=2023-02-01T19:22:51.1971578Z level=error message="Unable to load job file" error="unexpected end of JSON input" file=jobs/onion/1040.json
timestamp=2023-02-01T19:22:51.197174351Z level=error message="Failed to initialize module" error="unexpected end of JSON input" module=filedatastore
timestamp=2023-02-01T19:22:51.197189594Z level=error message="Aborting module launch due to initialization error" error="unexpected end of JSON input"
timestamp=2023-02-01T19:22:51.197417894Z level=info message="Terminating Modules"
timestamp=2023-02-01T19:28:54.061273291Z level=info message="Version Information" buildTime=2023-01-12T07:17:58Z version=2.3.200
timestamp=2023-02-01T19:28:54.061433041Z level=info message="Adding new preprocessor" priority=0 type=*web.BasePreprocessor
timestamp=2023-02-01T19:28:54.061469755Z level=info message="Launching modules"
timestamp=2023-02-01T19:28:54.061482245Z level=info message="Initializing module" module=elastic
timestamp=2023-02-01T19:28:54.070161852Z level=info message="Initialized Elasticsearch Client" HostUrl=https://10.1.10.228:9200 Index=*:so-*,*:endgame-* InsecureSkipVerify=true Password=***** TimeoutMs=5m0s Username=so_elastic
timestamp=2023-02-01T19:28:54.070254855Z level=info message="Initializing module" module=influxdb
timestamp=2023-02-01T19:28:54.070681487Z level=info message="Initializing module" module=sostatus
timestamp=2023-02-01T19:28:54.070729189Z level=info message="Initializing module" module=statickeyauth
timestamp=2023-02-01T19:28:54.070750655Z level=info message="Adding new preprocessor" priority=100 type=*statickeyauth.StaticKeyAuthImpl
timestamp=2023-02-01T19:28:54.070784867Z level=info message="Initializing module" module=staticrbac
timestamp=2023-02-01T19:28:54.070803534Z level=info message="Initializing module" module=filedatastore
timestamp=2023-02-01T19:28:54.070950113Z level=error message="Syntax error reading json object" error="unexpected end of JSON input" offset=0
timestamp=2023-02-01T19:28:54.070979043Z level=error message="Unable to load job file" error="unexpected end of JSON input" file=jobs/onion/1040.json
timestamp=2023-02-01T19:28:54.070998994Z level=error message="Failed to initialize module" error="unexpected end of JSON input" module=filedatastore
timestamp=2023-02-01T19:28:54.071019467Z level=error message="Aborting module launch due to initialization error" error="unexpected end of JSON input"
timestamp=2023-02-01T19:28:54.071278509Z level=info message="Terminating Modules"
timestamp=2023-02-01T19:29:25.669955277Z level=info message="Version Information" buildTime=2023-01-12T07:17:58Z version=2.3.200
timestamp=2023-02-01T19:29:25.670176533Z level=info message="Adding new preprocessor" priority=0 type=*web.BasePreprocessor
timestamp=2023-02-01T19:29:25.670250322Z level=info message="Launching modules"
timestamp=2023-02-01T19:29:25.670268511Z level=info message="Initializing module" module=filedatastore
timestamp=2023-02-01T19:29:25.670426181Z level=error message="Syntax error reading json object" error="unexpected end of JSON input" offset=0
timestamp=2023-02-01T19:29:25.670455852Z level=error message="Unable to load job file" error="unexpected end of JSON input" file=jobs/onion/1040.json
timestamp=2023-02-01T19:29:25.670478295Z level=error message="Failed to initialize module" error="unexpected end of JSON input" module=filedatastore
timestamp=2023-02-01T19:29:25.670498543Z level=error message="Aborting module launch due to initialization error" error="unexpected end of JSON input"
timestamp=2023-02-01T19:29:25.670689865Z level=info message="Terminating Modules

Also tried issuing command so-soc-restart.  All this does it change the status from "MISSING" to "ERROR" when issuing command so-status.  Any ideas on a fix?

**Comment by reyesj2** [comment, marked as answer]
> timestamp=2023-02-01T18:44:56.28610788Z level=error message="Unable to load job file" error="unexpected end of JSON input" file=jobs/onion/1040.json

Try deleting that file. ```rm /nsm/soc/jobs/onion/1040.json``` and then restarting soc ```so-restart soc```

**Reply by jamestest11** [reply]
This worked thank you!
